learning_journal/__init__.py

```diff
@@ -5,7 +5,7 @@
 def main(global_config, **settings):
     """ This function returns a Pyramid WSGI application."""
     if "sqlalchemy.url" not in settings:
-        settings["sqlalchemy.url"] = os.environ["DATABASE_URL"]
+        settings["sqlalchemy.url"] = os.environ.get('DATABASE_URL', 'sqlite:///:memory:')
```

Comment:
you probably don't want to have your database url default to in-memory if there is no env var specified...  Think about how this affects persistance.